src/components/RefineSearch/AdvancedSearchCheckboxField.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  FormRow,
+  FormField,
+  CheckboxGroup,
+  Checkbox,
+} from "@nypl/design-system-react-components"
+import { materialTypeOptions } from "../../utils/advancedSearchUtils"
+import { searchAggregations } from "../../config/aggregations"
+
+const AdvancedSearchCheckboxField = ({
+  searchFormState,
+  handleCheckboxChange,
+  name,
+  label,
+}) => {
+  const options = {
+    format: materialTypeOptions,
+    location: searchAggregations.buildingLocation,
```

Comment:
The component looks good but just pointing out it's coupled with the data it will render. Since this component only renders on `/search/advanced` it's not a big deal but components should try to render it's props rather than pull in data directly. The way you set this component, it looks very reusable including outside the scope of the advanced search page if this direct import was removed. You could try to import the data at a higher level and pass it down.

Reply:
Ok! Also added a todo comment to standardize the refine search and advanced search checkboxes. they're virtually identical except for some styling.

Reply:
update - did the refactor to do away with the original refine search checkbox component (which was also coupled to the refine search panel in its own special way 🥲)